corpus/accounts/models.py

```diff
@@ -150,3 +151,86 @@ def save(self, *args, **kwargs):
 
     def __str__(self):
         return f"{self.user.first_name} {self.user.last_name} [{self.sig.name}]"
+
+
+class Core(models.Model):
+
+    POST_CHOICES = (
```

Comment:
As discussed on call, any reason for this? I think it'll be better to make another model or something called `Post` and have a `ForeignKey` linking to that table.

Reply:
right, would the mapping remain the same for the core posts then? Like rn I'm having a POST_CHOICES which maps every core post to a number, according to which the core posts are fetched and displayed. So can I move this exact logic to the Post model when I create it?